apps/marketplace/src/app/item/[songId]/page.tsx

```diff
@@ -29,63 +30,69 @@ const SingleSong: FunctionComponent<SingleSongProps> = ({ params }) => {
-              color={ (theme) => theme.colors.grey300 }
-              variant="subtitle2"
-            >
-              { songData?.isExplicit ? "Explicit" : null }
-            </Typography>
-          </Stack>
-          <Typography textAlign={ "left" } variant="subtitle1">
-            { songData?.description }
-          </Typography>
+          <Box mb={ [2, 2, 0] } mr={ [0, 0, 5] } width={ [240, 240, 400] }>
+            <SongCard
+              coverArtUrl={ songData?.coverArtUrl }
+              imageDimensions={ 480 }
+              isPlayable={ true }
+              price={ mockSongs[0].price }
+              // eslint-disable-next-line @typescript-eslint/no-empty-function
+              onCardClick={ () => {} }
+              // eslint-disable-next-line @typescript-eslint/no-empty-function
+              onPriceClick={ () => {} }
+              // eslint-disable-next-line @typescript-eslint/no-empty-function
+              onSubtitleClick={ () => {} }
+            />
+          </Box>
           <Stack
-            alignItems={ "center" }
-            direction={ "row" }
-            gap={ 1.5 }
-            justifyContent={ ["center", "center", "start"] }
+            gap={ [4, 4, 2.5] }
+            pt={ [0, 0, 1.5] }
+            textAlign={ ["center", "center", "left"] }
+            width={ ["100%", 440, 440] }
           >
-            <ProfileImage
-              height={ 40 }
-              src={ resizeCloudinaryImage(songData?.artist.profileImageUrl, {
-                height: 56,
-                width: 56,
-              }) }
-              width={ 40 }
-            />
-            <Typography variant="h4">
-              { songData?.artist.firstName } { songData?.artist.lastName }
+            <Stack gap={ 0.5 }>
+              <Typography variant="h3">{ songData?.title }</Typography>
+              <Typography
+                color={ (theme) => theme.colors.grey300 }
+                variant="subtitle2"
+              >
+                { songData?.isExplicit ? "Explicit" : null }
+              </Typography>
+            </Stack>
+            <Typography textAlign={ "left" } variant="subtitle1">
+              { songData?.description }
             </Typography>
+            <Stack
+              alignItems={ "center" }
+              direction={ "row" }
+              gap={ 1.5 }
+              justifyContent={ ["center", "center", "start"] }
+            >
+              <ProfileImage
+                height={ 40 }
+                src={ resizeCloudinaryImage(songData?.artist.profileImageUrl, {
+                  height: 56,
+                  width: 56,
+                }) }
+                width={ 40 }
+              />
+              <Typography variant="h4">
+                { songData?.artist.firstName } { songData?.artist.lastName }
+              </Typography>
+            </Stack>
           </Stack>
         </Stack>
-      </Stack>
+      </Container>
+
+      <MoreSongs />
+
+      <SimilarSongs />
```

Comment:
There are skeletons for the individual songs as they load in but should we also have a larger block skeleton in `ItemSkeleton.tsx` to load with the top half of the Single Item/Song data?